Makefile

```diff
@@ -3,23 +3,64 @@
 ANOLIS = anolis
 TIDY-HTML5 = tidy-html5
 
+# This hacky set of shell commands should return the URL of the latest
+# version of the spec published in /TR/ space. This URL is used to compute the
+# "Previous version" link automatically.
+# TODO: Rather use the "Web API" that W3C will soon expose once it is available
+PUBLISHED-URL=$(shell curl -s http://www.w3.org/TR/presentation-api/ \
+	| grep "This version" --after-context=6 \
+	| grep "<a href" \
+	| sed "s/^.*<a href=\"//" \
+	| sed "s/\".*//")
+
+
+# Default rule generates the Editor's Draft
 all: tidy index.html xrefs/presentation.json
```

Comment:
It seems like the dependencies could be written this way to reduce a bit of duplication:

all: index.html xrefs/presentation.json
index:html: tidy ../xref
tidy: Overview.src.html tidyconfig.txt

That makes it clearer (IMO) that any change to Overview.src.html will trigger tidy, followed by Anolis to generate index.html.